client/ayon_core/tools/publisher/control.py

```diff
@@ -1662,6 +1664,16 @@ def current_task_name(self):
 
         return self._create_context.get_current_task_name()
 
+    @property
+    def current_project_settings(self):
```

Comment:
I would make this private `_current_project_settings`.

Reply:
I was following other `current_folder_path`, ` current_task_name`. Not really sure why these are as properties and `current_project_settings` should be private.

Reply:
Those properties are public api of controller, `current_project_settings` is used as private property of the controller.